### PR TITLE
Clean up and organize deploy

### PR DESCRIPTION
**Pull request by Azathotha**

### CODE REVIEW

File: index.html
Comment (dyc-github) [inline index.html:1]: Move index.html to src not at the top level.

File: src/json/note.json
Comment (dyc-github) [inline src/json/note.json:1]: This is up to you but I think we can just delete note.json as its never used.

File: src/api/API.js
Comment (dyc-github) [inline src/api/API.js:1]: I made a new task for this but I think we should refactor scripts.js to follow SRP more strictly closer to how this api file is structured (although it shouldn't be called API). For now though this is fine.

File: index.html
Comment (dyc-github) [reply index.html:1]: There is a conflict with the other index.html but feel free to rename it and just assign me a task to fix it.

Comment (dyc-github) [reply index.html:1]: If you can make script in the head and run onDOMContentLoaded or with async that would likely be better but its a nitpick.

**Review by dyc-github — CHANGES_REQUESTED** [review]
Your PR fails the linter and I left a few comments on what else to fix. Most are small or nitpicky but still worth looking at.

**Review by symsoph — APPROVED** [review]
Not sure if the task David referred to - about refactoring script.js - is resolved. But everything seems to be okay

**Review by dyc-github — APPROVED** [review]

**Pull request merged**